### PR TITLE
semantic cerner alert and check axe on test

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Screenshots
Capture

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
can you add a screenshot to this PR?

File: src/applications/personalization/dashboard/components/cerner-widgets.js
Comment (tmitchellgcio) [reply src/applications/personalization/dashboard/components/cerner-widgets.js:1]: yep sorry didn't realize that. I will get right on that

**Comment by tmitchellgcio** [comment]
> can you add a screenshot to this PR?

yes, but I'm waiting on confirmation that the screenshots I posted are what @tressaellen intended. It's on this task https://github.com/department-of-veterans-affairs/va.gov-team/issues/28575

**Comment by erikphansen** [comment]
@tressaellen are you the authority and which parts of the text should be bold and which shouldn't? If so, please review Taylor's screenshot above. I think the 2 instances of "For" should _not_ be bold. And "If you have trouble..." should also not be bold, but please confirm.

**Comment by tmitchellgcio** [comment]
Updated screenshot

**Review by erikphansen — APPROVED** [review]

File: src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js
Comment (erikphansen) [inline src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js:15]: just FYI you could have also done:
```
to.contain(`For ${facilityNames[0]}:`)
```
or something like:
```
to.match(new RegExp(`^For ${facilityNames[0]}:`))
```

**Review by erikphansen — APPROVED** [review]

File: src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js
Comment (tmitchellgcio) [reply src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js:15]: Good to know. Thank you. Would you advise I change this before merging?

**Pull request merged**
